scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -0,0 +1,155 @@
+
+### Step 6. Configure permissions in GCP
+
+Your Terra proxy group needs to be granted specific roles in GCP so Terra can create, read, and write to BigQuery tables as part of the GVS workflow.
+
+Grant your proxy group the BigQuery Data Editor, BigQuery Job User, and BigQuery Read Session User roles on the Google Project that you created in Step 3 (above) that contains your BigQuery dataset. For step-by-step instructions, see the Google Cloud documentation article, [Manage access to projects, folders, and organizations](https://cloud.google.com/iam/docs/granting-changing-revoking-access#grant-single-role).
+
+If you’ve done this correctly, you should see your Terra proxy group listed in the table of principals along with the roles you granted it.
+
+### Step 7. Clone the GVS beta workspace
+
+The GVS beta workspace in Terra is read-only, so you’ll need to clone the workspace to create a copy where you can upload your own data and run the workflow. Clone the workspace using the billing project you created in Step 2 (above) by following the instructions in the article [Make your own project workspace](https://support.terra.bio/hc/en-us/articles/360026130851).
+
+## Running the workflow
+
+The workflow in the GVS beta workspace is pre-configured to use 10 sample GVCF files as a sample set in the sample_set data table in the workspace Data tab.
+
+The workflow is configured to call this input from the data table. To run:
+
+1. Select the workflow from the Workflows tab.
+1. In the configuration page, select the sample_set entity in Step 1.
+1. Select the sample_set dataset in Step 2.
+1. Configure the workflow inputs.
+    1. Enter a name for the callset as a string with the format “*CALLSET_NAME*” for the `callset_identifier` variable. This string is used as to name several variables and files and should begin with a letter. Valid characters include A-z, 0-9, “.”, “,”, “-“, and “_”.
+    1. Enter the name of the BigQuery dataset you created in Step 4 of Setup as a string with the format “*DATASET_NAME*” for the `dataset_name` variable.
+    1. Enter the name of the GCP project you created in Step 3 of Setup as a string with the format “*PROJECT_NAME*” for the `project_id` variable.
+1. Save the workflow configuration.
+1. Run the workflow.
+
+To run the GVS workflow on your own sample data, follow the instructions in the tutorial, [Upload data to Terra and run the GVS workflow](./run-your-own-samples.md).
+   
+### Important configuration notes
+
+By default, the workflow is set up to write outputs to the workspace Google bucket. If you want to write the outputs to a different cloud storage location, you can specify the cloud path in the `extract_output_gcs_dir` optional input in the workflow configuration. 
+
+### Time and cost estimates
+Below is an example of the time and cost of running the workflow.
+
+| Number of Samples | Wall Clock Time | Cost $ | Cost per Sample |
+| ---  | --- | --- | --- |
+| 10 | 04:30:00 | $0.84 | ~$0.08 |
```

Comment:
10 samples took 4.5 hours and 10k samples only took an hour longer? Interesting!

Reply:
We weren't running the unified WDL---so I'm a lil suspect of the numbers. I'm going to do a re-run of the 10k when I add back in the cost and timing info